src/app/component/product-details/product-details.component.html

```diff
@@ -0,0 +1,92 @@
+<div class="hero is-fullheight">
+    <div class="hero-body">
+        <div class="container">
+            <div class="level">
+                <div class="level-left">
+                    <div class="level-item">
+                        <p class="title is-2">
+                            Product: {{product.name}}
+                        </p>
+                    </div>
+                </div>
+
+                <div class="level-right">
+                    <div class="level-item" *ngIf="isCustomer()">
+                        <button class="button action-button is-large is-info" (click)="addToCart()">
+                            <span class="icon">
+                                <i class="fas fa-plus"></i>
+                                <i class="fas fa-shopping-cart"></i>
+                            </span>
+                        </button>
+                    </div>
+                    <div class="level-item" *ngIf="isAdmin()">
```

Comment:
It will be better to not have a method in the ngIf since it brings a lot of overhead in rendering. Better to have a variable inside the component and use that in the html